.github/workflows/plugins-ci-mongo.yml

```diff
@@ -23,6 +23,11 @@ on:
         required: false
         default: false
         type: boolean
+      fail-fast:
+        description: 'Set the fail-fast strategey setting.'
+        required: false
+        default: true
```

Comment:
default should be please false.

Reply:
I can change it to that, but the platform default is `true` so defaulting to `false` would be a behavior change from what it's doing now.

Reply:
Ah right. 👍